package/yaft/package

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(yaft)
+pkgdesc="Framebuffer terminal emulator with an on-screen touch keyboard"
+url=https://github.com/timower/rM2-stuff/tree/master/apps/yaft
+pkgver=0.0.1-1
+timestamp=2020-10-27T12:02Z
+section="admin"
+maintainer="Timothy Werquin"
+license=GPL-3.0
```

Comment:
If I’m not mistaken this should be MIT.
```suggestion
license=MIT
```

Reply:
I re-licensed the code at GPLv3 as my whole repo uses GPL.

Reply:
Also, the header of every file has this included comment:

> /* See LICENSE for licence details. */

Reply:
Whoops, I forgot to remove that file.